Update how-to-apply-for-teacher-training.md (#1788)
DFE-Digital/get-into-teaching-app: app/views/content/blog/how-to-apply-for-teacher-training.md

### 4. Apply for teacher training

> You're now ready to submit your application! You can apply via the new GOV.UK service called [Apply for teacher training.](https://www.gov.uk/apply-for-teacher-training).

Reviewer: ```suggestion
You're now ready to submit your application! You can apply using a GOV.UK service called [Apply for teacher training](https://www.gov.uk/apply-for-teacher-training).
```
